packages/apps/project/src/components/templates/ActivityListView/ActivityListView.tsx

```diff
@@ -94,13 +96,10 @@ const ActivityListView = () => {
 
   const onRefresh = useCallback(() => {
     setRefreshing(true);
-    setStartDate(() => {
-      const newDate = new Date();
-      newDate.setDate(newDate.getDate() + 1);
-      return newDate;
-    });
-    setDataList([]);
-    fetchActivityData(new Date()).finally(() => setRefreshing(false));
+    const newDate = new Date();
+    newDate.setDate(newDate.getDate() + 1);
+    setStartDate(newDate);
```

Comment:
As you set startDate here, it trigger the useEffect which call fetchActivityData without reset at true so the refresh issue is still here